README.md

```diff
@@ -424,6 +426,14 @@ List of the high level principles that a product /development team must adhere t
 
 What are the links of the supporting systems?
 
+#### Tracing Change events and requests Correlation Id
+
+  To be able to track a change event and the change request it can become across systems a common id field is present on logs related to each event. The id is generate in `Profile Editor` (NHS UK) which is then assigned to the `correlation-id` header of the request send to our (DoS Integration) endpoint, for a given change event. The `correlation-id` header is then used through the handling of the change event in `DoS Integration`.
```

Comment:
generated instead of generate

Also the last sentence could be changed to this 
```
Then `correlation_id` header is then used through the handling of the change event in `DoS Integration`.```